manifests/udev/rule.pp

```diff
@@ -23,8 +23,8 @@
 #   The literal udev rules you want to deploy
 #
 define systemd::udev::rule (
-  Array                             $rules,
-  Enum['present', 'absent', 'file'] $ensure                  = 'present',
+  Array                             $rules                   = [],
```

Comment:
I changed this so it always has a value. if you use the defined resource to remove a udev rule you now only have to set `$ensure` to absent. Without this change you also would need to set `$rules`